userguide/virtualmachines.rst

```diff
@@ -514,47 +514,47 @@ see the same environment.
 `Rancher <https://rancher.com/>`__
 is a web-based tool for managing Docker containers.
 
-%brand% runs the Rancher web interface within the Docker Host.
+%brand% runs the Rancher web interface within the |dockerhost|.
 
 
-.. index:: Docker Host Requirements
-.. _Docker Host Requirements:
+.. index:: |dockerhost| Requirements
+.. _|dockerhost| Requirements:
 
-Docker Host Requirements
-~~~~~~~~~~~~~~~~~~~~~~~~
+|dockerhost| Requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The system BIOS **must** have virtualization support enabled for a
-Docker Host to run properly after installation. On Intel systems this is
-typically an option called *VT-x*. AMD systems generally have an *SVM*
-option.
+|dockerhost| to run properly after installation. On Intel systems this
+is typically an option called *VT-x*. AMD systems generally have an
+*SVM* option.
 
-20 GiB of storage space is required for the Docker Host.
+20 GiB of storage space is required for the |dockerhost|.
 
 For setup, the :ref:`SSH` service must be enabled.
 
-The Docker Host requires 2 GiB of RAM while running.
+The |dockerhost| requires 2 GiB of RAM while running.
 
 
-.. index:: Docker Host
-.. _Creating Docker Host:
+.. index:: |dockerhost|
+.. _Creating |dockerhost|:
 
-Creating Docker Host
-~~~~~~~~~~~~~~~~~~~~
+Creating |dockerhost|
+~~~~~~~~~~~~~~~~~~~~~
 
 :numref:`Figure %s <vms_add_docker_fig>` shows the Wizard that appears
 after going to
 :menuselection:`Virtual Machines`,
```

Comment:
```suggestion
|ui-add| :menuselection:`--> Virtual Machines`
```

Reply:
I had to break the navigation steps apart because I'm pointing to a screen within the wizard.